src/amber/cli/templates/app/config/environments/development.yml.ecr

```diff
@@ -20,7 +20,7 @@ secrets:
   when "mysql" -%>
   database: mysql://root@localhost:3306/<%= database_name_base %>_development
 <%when "pg" -%>
-  database: postgres://postgres:@pg:5432/<%= database_name_base %>_development
+  database: postgres://postgres:@localhost:5432/<%= database_name_base %>_development
```

Comment:
These were not consistent with the current `config/database.yml`. Two things to note.

- We should get rid of `config/database.yml`
- Be able to load environment dynamically `Amber.env.load(:test)`